main/main.cpp

```diff
@@ -1009,6 +1009,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	bool use_custom_res = true;
 	bool force_res = false;
 	bool delta_smoothing_override = false;
+	bool gdextension_reload_in_game = false;
```

Comment:
This should fix the `unused-variable` errors
```suggestion
#ifdef TOOLS_ENABLED
	bool gdextension_reload_in_game = false;
#endif // TOOLS_ENABLED
```